lib/src/spec/rest/ably_base.dart

```diff
@@ -27,13 +23,15 @@ abstract class AblyBase {
   static PresenceMessageStatic presenceMessage;
   Auth auth;
   Push push;
+
   Future<PaginatedResultInterface<Stats>> stats([Map<String, dynamic> params]);
-  Future<HttpPaginatedResponse> request({
-    @required String method,
-    @required String path,
-    Map<String, dynamic> params,
-    dynamic body,
-    Map<String, String> headers
-  });
+
+  Future<HttpPaginatedResponse> request(
+      {@required String method,
+      @required String path,
+      Map<String, dynamic> params,
+        Object body,
+      Map<String, String> headers});
```

Comment:
Even though it's formatted vertically, the formatting with trailing comma is a bit nicer IMHO